task/08-objects-tasks.js

```diff
@@ -23,7 +23,10 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+    this.width = width;
+    this.height = height;
+
+    Rectangle.prototype.getArea = () => this.width * this.height;
```

Comment:
Please,   move this logic outside Rectangle function, because everytime function Rectange be invoked - every time prototype will be changed
But remember about context(this)